src/classes/UTIL_CustomSettingsFacade.cls

```diff
@@ -548,7 +548,7 @@ public without sharing class UTIL_CustomSettingsFacade {
             hs.npo02__Household_Rules__c = HH_Households.NO_HOUSEHOLDS_PROCESSOR;
         }
 
-        hs.npo02__Household_Member_Contact_Role__c = label.npo02.Household_Member_Contact_Role;
+        hs.npo02__Household_Member_Contact_Role__c = CAO_Constants.HH_MEMBER_CONTACT_ROLE;
```

Comment:
Yes. Since these all refer to Picklist values, the value for this field has to be "Household Member" since that is our packaged default. This label was also one that _should_ not have translations in our C&O package, however due to the issue documented where translations removed from our repository were not actually removed from the package itself there is a possibility that customers may actually have translations for this old label even though they shouldn't. Changing the label reference to an Apex Constant reference eliminates that potential issue.